dependencies.props

```diff
@@ -17,12 +17,6 @@
     <CurrentRefXmlPath>$(MSBuildThisFileFullPath)</CurrentRefXmlPath>
   </PropertyGroup>
 
-  <!-- Package versions used as toolsets -->
```

Comment:
Is this file used at all anymore? Should just be 2.x stuff.

Reply:
I think you're right, did some other cleanup of ProdCon stuff too.